### PR TITLE
Display PNC query data in PNC details panel

### PR DESCRIPTION
**Pull request by tausifpatel**
- Created a PNC panel to display PNC data

Screenshots:
Screenshot 2024-07-11 at 4 26 36 PM
Screenshot 2024-07-11 at 4 26 50 PM

### CODE REVIEW

File: cypress/component/PncDetails.cy.tsx [hunk 1]
```diff
@@ -0,0 +1,172 @@
...
+    const courtCase = {
+      aho: {
+        PncQuery: pncQueryData,
+        PncQueryDate: "2024-07-10T00:00:00.000Z"
+      }
+    } as unknown as DisplayFullCourtCase
+
+    cy.mount(
+      <CourtCaseContext.Provider value={[{ courtCase, amendments: {}, savedAmendments: {} }, () => {}]}>
+        <CurrentUserContext.Provider value={{ currentUser }}>
```
Comment (richrace) [inline cypress/component/PncDetails.cy.tsx:81]: Small thing: In the `court-cases/[courtCaseId]/index.tsx` page the provider is the other way round. E.g.

```jsx
<CurrentUserContext.Provider value={currentUserContext}>
  <CourtCaseContext.Provider value={courtCaseContext}>
  </CourtCaseContext.Provider>
</CurrentUserContext.Provider>
```

File: src/features/CourtCaseDetails/Sidebar/Pnc/PncDetails.tsx [hunk 2]
```diff
@@ -0,0 +1,55 @@
+import { useCourtCase } from "context/CourtCaseContext"
```
Comment (richrace) [inline src/features/CourtCaseDetails/Sidebar/Pnc/PncDetails.tsx:1]: Does the PNC folder need to be removed?

File: src/features/CourtCaseDetails/Sidebar/Pnc/PncOffence.tsx [hunk 3]
```diff
@@ -0,0 +1,85 @@
+import useFormattedDate from "hooks/useFormattedDate"
```
Comment (richrace) [inline src/features/CourtCaseDetails/Sidebar/Pnc/PncOffence.tsx:1]: Does the PNC folder need to be removed?

File: src/features/CourtCaseDetails/Sidebar/PncDetails/PncOffenceDetails.tsx [hunk 4]
```diff
@@ -0,0 +1,85 @@
+import useFormattedDate from "hooks/useFormattedDate"
+
+interface Props {
```
Comment (richrace) [inline src/features/CourtCaseDetails/Sidebar/PncDetails/PncOffenceDetails.tsx:3]: Should: Use a better name than rather generic props.

File: src/features/CourtCaseDetails/Sidebar/PncDetails/Disposal.tsx [hunk 5]
```diff
@@ -0,0 +1,63 @@
+import ConditionalRender from "components/ConditionalRender"
+import { DisposalDetails, DisposalHeader, DisposalText } from "./Disposal.styles"
+// import { formatDisplayedDate } from "utils/date/formattedDate"
+import { Details } from "govuk-react"
+import { isEmpty } from "lodash"
+import useFormattedDate from "hooks/useFormattedDate"
+
+interface Props {
```
Comment (richrace) [inline src/features/CourtCaseDetails/Sidebar/PncDetails/Disposal.tsx:8]: Should: Use a better name than rather generic props.

**Review by C-gyorfi — APPROVED** [review]
LGTM, after we deploy can we confirm if the displayed time format is consistent with the legacy application?

**Pull request merged**